reports/proofofplay-report-form.twig

```diff
@@ -80,6 +80,14 @@
                             {% set title %}{% trans "Time" %}{% endset %}
                             {{ inline.time("statsToDtTime", title, "00:00", "", "stats-to-dt-time") }}
 
+                            {% set title %}{% trans "Group By" %}{% endset %}
+                            {% set options = [
+                                { id: "", name: "" },
```

Comment:
Can we call this option "Display" ?

Reply:
Yup! Added `display` as a default option. Thank you for the review!